sdk/tests/deqp/modules/shared/glsLifetimeTests.js

```diff
@@ -992,11 +992,11 @@ glsLifetimeTests.OutputAttachmentTest.prototype.iterate = function() {
     var element = elementType.gen();
 
     attacher.initAttachment(0, element);
+    this.m_outputAttacher.drawAttachment(element, refSurface);
     attacher.attach(element, container);
```

Comment:
The idea was that it would unbind the buffer from TRANSFORM_FEEDBACK_BUFFER to avoid the collision where the same buffer is bound to both the ARRAY_BUFFER and TRANSFORM_FEEDBACK_BUFFER binding points.

Maybe I'm thinking about the problem incorrectly, but rather than reordering the drawAttachment/setupContainer calls, we should add more hooks to allow the state to be compatible with WebGL's semantics.